azkaban-web-server/src/test/java/azkaban/imagemgmt/services/ImageTypeServiceImplTest.java

```diff
@@ -75,9 +83,48 @@ public void testCreateImageType() throws Exception {
     Assert.assertEquals(100, imageTypeId);
   }
 
+  @Test
+  public void testGetAllImageTypesWithOwnerships() throws Exception {
+    List<ImageType> imageTypes = getImageTypeList();
+    ImageTypeDTO expected = getImageTypeDTO();
+    when(this.imageTypeDao.getAllImageTypesWithOwnerships()).thenReturn(imageTypes);
+    List<ImageTypeDTO> imageTypeDTOs = this.imageTypeService.getAllImageTypesWithOwnerships();
+    assert imageTypeDTOs.size() == 1;
+    assertObjectsAreEqual(imageTypeDTOs.get(0), expected);
+  }
+
+  @Test
+  public void testFindImageTypeWithOwnershipsById() throws Exception {
+    String id = "1";
```

Comment:
Have you Installed the Intellij's Save Actions plugin mentioned in https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md#style-guides in your IDE?

Reply:
It is installed and enabled, yes.